homeassistant/config.py

```diff
@@ -681,10 +690,14 @@ def format_homeassistant_error(
 
 @callback
 def format_schema_error(
-    ex: vol.Invalid, domain: str, config: dict, link: str | None = None
+    hass: HomeAssistant,
```

Comment:
Be sure to mention this addition when talking to the custom integration authors about the change in the signature.